WhiteDentalClinic/WhiteDentalClinic.Api/Controllers/MedicalServiceController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using WhiteDentalClinic.Application.Models.Dentist;
+using WhiteDentalClinic.Application.Models;
+using WhiteDentalClinic.Application.Models.MedicalServiceModel;
+using WhiteDentalClinic.Application.Services.Interfaces;
+
+namespace WhiteDentalClinic.Api.Controllers
+{
+    public class MedicalServiceController : ApiController
```

Comment:
Consistency is quite important when coding. Some of your controllers are name plural and some of them singular -> this is a sign that you need to consider one approach out of the two and use that everywhere. In this specific scenario it's clear that they need to be pluralized as this is a best practice but even when a best practice is not clearly specified, similar components should behave/be named similar.